crates/goose-cli/src/commands/session.rs

```diff
@@ -27,8 +27,50 @@ pub fn remove_sessions(sessions: Vec<SessionInfo>) -> Result<()> {
     Ok(())
 }
 
-pub fn handle_session_remove(id: String, regex_string: String) -> Result<()> {
-    let sessions = match get_session_info(SortOrder::Descending) {
+fn prompt_interactive_session_selection(sessions: &[SessionInfo]) -> Result<Vec<SessionInfo>> {
+    if sessions.is_empty() {
+        println!("No sessions available to select.");
+        return Ok(vec![]);
+    }
+
+    let mut selector = multiselect(
+        "Select sessions to delete (use spacebar, Enter to confirm, Ctrl+C to cancel):",
```

Comment:
wording is confusing. if it's what I think you are saying, then perhaps this?
```suggestion
        "Select sessions to delete (use spacebar or Enter to confirm, Ctrl+C to cancel):",
```